enclave_wrangler/config.py

```diff
@@ -145,14 +145,14 @@
         'rid': 'ri.foundry.main.dataset.f945409a-37f1-402f-a840-29b6bd675cb0',
         'column_names': ["codeset_id", "approx_distinct_person_count", "approx_total_record_count"],
         'sort_idx': ['codeset_id'],
-        'dataset_groups': ['vocab']
+        'dataset_groups': ['counts']
```

Comment:
Because the counts tables and the vocab tables [are updated days apart](https://github.com/jhu-bids/TermHub/pull/502#discussion_r1285261401), it seems most advisable to run this [cronjob daily](https://github.com/jhu-bids/TermHub/pull/502#discussion_r1285140413), rather than weekly, and have it only do anything when it sees if counts or vocab tables are outdated.